source/commands/create-bounty-board.js

```diff
@@ -44,7 +44,19 @@ module.exports = new CommandWrapper(customId, "Create a new bounty board forum c
 			scoreboardPost.pin();
 			guildProfile.scoreId = scoreboardPost.id;
 
-			//TODO create posts for existing bounties
+			database.models.Bounty.findAll({ where: { guildId: interaction.guildId, state: "open" } }).then(bounties => {
```

Comment:
Very niche concern here: we should try to do some sorting here on the query to have a consistent order by the time we loop through and make bounty board posts.

Reply:
Added in https://github.com/Imaginary-Horizons-Productions/BountyBot/pull/23/commits/18e3edb67d7e1990d7baa5575428a74b3887fe5f